.github/actions/workflow-vars/action.yaml

```diff
@@ -125,6 +129,20 @@ runs:
       # The var is needed in some non-shell steps.
       echo "ingress-host=${TEST_INGRESS_HOST}" | tee -a $GITHUB_OUTPUT
 
+      ## Setting values for the external infra
+      echo "ORCHESTRATION_INDEX_PREFIX=$GITHUB_WORKFLOW_JOB_ID-orchestration" | tee -a $GITHUB_ENV
+      echo "OPTIMIZE_INDEX_PREFIX=$GITHUB_WORKFLOW_JOB_ID-optimize" | tee -a $GITHUB_ENV
+      echo "OPERATE_INDEX_PREFIX=$GITHUB_WORKFLOW_JOB_ID-operate" | tee -a $GITHUB_ENV
+      echo "TASKLIST_INDEX_PREFIX=$GITHUB_WORKFLOW_JOB_ID-tasklist" | tee -a $GITHUB_ENV
+      echo "FLOW=$FLOW" | tee -a $GITHUB_ENV
+      keycloakRealm="${GITHUB_WORKFLOW_JOB_ID}-realm"
+      export KEYCLOAK_REALM=$keycloakRealm
+      echo "KEYCLOAK_REALM=$KEYCLOAK_REALM" | tee -a $GITHUB_ENV
+      echo "KEYCLOAK_TOKEN_URL_INT=https://${TEST_INGRESS_HOST}/auth/realms/camunda-platform/protocol/openid-connect/token" | tee -a $GITHUB_ENV
```

Comment:
Have we checked with QA team to see if their workflows visit keycloak's admin portal at all?

they might validate that identity creates data in keycloak, in which case, their tests would break from this overall  change, I think.